pynetbox_data_uploader/lib/user_methods/csv_to_netbox.py

```diff
@@ -24,63 +27,96 @@ def __init__(self, url: str, token: str):
         :param token: The Netbox auth token.
         """
         self.netbox = NetboxConnect(url, token).api_object()
-        self.format_dict = FormatDict(self.netbox)
         self.exist = NetboxCheck(self.netbox)
         self.create = NetboxCreate(self.netbox)
+        self.query_dataclass = QueryDataclass(self.netbox)
 
-    def read_csv(self, file_path) -> List:
+    @staticmethod
+    def check_file_path(file_path: str):
+        """
+        This method checks if the filepath is valid. Raises an exception if it's invalid.
+        :param file_path: The path to the csv file.
+        """
+        print("Checking filepath...")

```

Comment:
Consider whether these should be logged to stderr instead of printed

Reply:
These are printed so the user knows what is happening when the script is running because it can take a while. In this case should it not be `sys.stdout.write()` as they are not errors.